trackintel/io/file.py

```diff
@@ -10,6 +10,9 @@ def read_positionfixes_csv(*args, **kwargs):
     builds a geopandas GeoDataFrame. This also validates that the ingested data
     conforms to the trackintel understanding of positionfixes (see 
     :doc:`/modules/model`). 
+    
+    In addition to the pandas read_csv keyword arguments, this function provides to rename columns directly while importing
+    by specifying the columnnames in a dictionary, see second example.
```

Comment:
I am wondering if we need to explicitly write out the `column` as a parameter and describe how to use it (mainly just original col as key and standard col names as value), an example of comment see [here ](https://github.com/mie-lab/trackintel/blob/master/trackintel/preprocessing/positionfixes.py).

Reply:
Yes I thought about it too, but was not sure if this is "clean" to explicitly describe keyword arguments. But as this is the only one that is used by our function, it makes sense to add this.